src/monster.cpp

```diff
@@ -458,12 +456,88 @@ int monster::print_info(WINDOW* w, int vStart, int vLines, int column) const
 
 std::string monster::extended_description() const
 {
-    // @todo Add tons of info here
-    std::string ret = string_format( _ ( "This is a %s" ), name().c_str() );
-    ret += "\n--\n";
+    std::ostringstream ss;
+
+    nc_color color = c_white;
+    std::string attitude = "";
+    get_Attitude( color, attitude );
+    attitude = get_tag_from_color( color ) + attitude;
+
+    ss << string_format( _( "This is a %s %s" ), name().c_str(), attitude.c_str() ) << std::endl;
+    if( !get_effect_status().empty() ) {
+        ss << string_format( "<stat>It is %s</stat>", get_effect_status().c_str() ) << std::endl;
+    }
+
+    ss << "--" << std::endl;
     auto hp_bar = hp_description( hp, type->hp );
-    ret += get_tag_from_color( hp_bar.second ) + hp_bar.first;
-    return replace_colors( ret );
+    ss << get_tag_from_color( hp_bar.second ) << hp_bar.first << std::endl;
+
+    ss << "--" << std::endl;
+    ss << string_format( "<dark>%s</dark>", type->description.c_str() ) << std::endl;
+    ss << "--" << std::endl;
+
+    std::vector<std::string> senses;
+    if( type->has_flag( m_flag::MF_HEARS ) ) {
+        senses.emplace_back( "hearing" );
```

Comment:
This flags defined in code so to make it in JSON It's needed to move those flags to JSON first.